Library/Homebrew/PATH.rb

```diff
@@ -15,7 +15,6 @@ class PATH
   Element = T.type_alias { T.nilable(T.any(Pathname, String, PATH)) }
   private_constant :Element
   Elements = T.type_alias { T.any(Element, T::Array[Element]) }
-  private_constant :Elements
```

Comment:
This is useful in typing the `Kernel.which` extension. (`Kernel` probably shouldn't reach into `PATH` like this, but it also shouldn't depend on `PATH` at all 🤷)